test/factories/patient.rb

```diff
@@ -3,7 +3,7 @@
     name 'New Patient'
     sequence(:primary_phone, 100) { |n| "127-#{n}-1111" }
     line 'DC'
-    spanish false
+    language 'English'
```

Comment:
I think we can get rid of this since we're assuming that nil == english, but I'll fix it direct since I think this is an important PR and I want to get it shipped.